### PR TITLE
Update Onboarding Bracket

### LINKED ISSUE #3865: [BUG]: Broken images in onboarding
### Describe the bug

Page 5 of onboarding tries to display two broken or missing images.

### Steps to Reproduce

1. Start from a fresh install, or reset onboarding
2. Advance to page 5

### Expected Behavior

2 images related to the shelf bracket are displayed.

### Screenshots and Recordings

image

### Desktop OS

Ubuntu

### Browser

_No response_

### Version

v0.25.1

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by jgomez720**
closes #3865

### CODE REVIEW

**Comment by jgomez720** [comment]
@greg-kcio I couldn't replicate the broken images problem, both locally and in our current deployment (app.zoo.dev). When Vercel deploys can you double check that they work?

**Comment by greg-kcio** [comment]
> @greg-kcio I couldn't replicate the broken images problem, both locally and in our current deployment (app.zoo.dev). When Vercel deploys can you double check that they work?

Will do. Maybe it's only an issue for the Linux app. Could be an incompatible path.

**Comment by greg-kcio** [comment]
Web app preview looks correct, except for KCL syntax highlighting being broken.

image

**Review by greg-kcio — APPROVED** [review]

**Comment by jgomez720** [comment]
sweet. Thank you!

**Comment by jgomez720** [comment]
@lf94 I'm far out of the electron-tests loop. Are the tests that are failing something I can fix?

**Comment by lf94** [comment]
It looks like the failing test uses text based on the old example, should be an easy fix you want to try first :slightly_smiling_face:

**Comment by jgomez720** [comment]
Taking a crack at it

**Comment by lf94** [comment]
Seems the test failed but in the same area, you may have missed something

**Comment by franknoirot** [comment]
I think the error is just sometimes not ending up with the message `Unexpected token:` depending on where the cursor ends up being, just buttoning the test up then this will be g2g

**Comment by lf94** [comment]
Thanks @franknoirot I had told Josh I'd take it after he'd tried a few times but I've been deep in the WebRTC engine side for a moment

**Comment by franknoirot** [comment]
No sweat sorry I missed the earlier GH comments, I agree this is a trust melter. I'm currently looking at how to turn off workflow run notifications in GH, they absolutely flood my inbox and I miss important stuff like this 💢

File: e2e/playwright/code-pane-and-errors.spec.ts [hunk 1]
```diff
@@ -65,6 +65,8 @@ const extrude001 = extrude(5, sketch001)`
   test('Opening and closing the code pane will consistently show error diagnostics', async ({
     page,
   }) => {
+    await page.goto('http://localhost:3000')
```
Comment (jgomez720) [inline e2e/playwright/code-pane-and-errors.spec.ts:68]: I was trying things at midnight last night 😆 feel free to delete lol

Comment (franknoirot) [reply e2e/playwright/code-pane-and-errors.spec.ts:68]: Aw dang sorry man!

Comment (franknoirot) [reply e2e/playwright/code-pane-and-errors.spec.ts:68]: That had no impact on the test so I left it for now, it's very easy to clean up on another pass, we're talking like an extra 400ms on a test run.

**Pull request merged**